lib/state/ui/reducer.ts

```diff
@@ -175,7 +187,7 @@ const note: A.Reducer<T.NoteEntity | null> = (state = null, action) => {
     case 'App.emptyTrash':
     case 'App.showAllNotes':
     case 'App.selectTrash':
-    case 'App.selectTag':
+    case 'OPEN_TAG':
```

Comment:
this is a job for another day/PR but I don't think we need all these anymore since `FILTER_NOTES` takes care of them. maybe it's still best. what's odd is setting it to `null` and then immediately setting it to some other note